src/dss/list.py

```diff
@@ -21,34 +24,54 @@ def list_notebooks(lightkube_client: Client, wide: bool = False) -> None:
                                Defaults to False.
     """
     try:
-        deployments = lightkube_client.list(Deployment, namespace=DSS_NAMESPACE)
+        deployments = list(
+            lightkube_client.list(
+                Deployment,
+                namespace=DSS_NAMESPACE,
+                labels={NOTEBOOK_LABEL: lightkube.operators.exists()},
+            )
+        )
     except ApiError as e:
         logger.error(f"Failed to list notebooks: {e}")
         return
 
-    notebook_list = []
+    if not deployments:
+        # Use stdout for table so the output can be piped properly
+        print("No notebooks found in dss")
+        return
+
+    # Create a PrettyTable object
+    table = PrettyTable()
+    table.field_names = ["Name", "Image", "URL"]
+    table.border = False
+    table.align = "l"  # Align the text to left
+
+    # Adjust column widths based on the output destination
+    if sys.stdout.isatty() and not wide:
+        # Output is to a terminal and not in wide mode
+        table._max_width = {"Name": 26, "Image": 30, "URL": 24}
```

Comment:
Can we increase the URL width here?  Name and Image aren't so bad when they're truncated, but URL is kinda useless.  The max width possible is 29 or 30 char, although since we set the port and it wont be https I think 26 will catch everything we need

Reply:
Ok I see also that if anything is wider than the column it gets wrapped rather than truncated, so the content is still available either wya.  And there is a `--wide` option.  So if you want to keep it as is that is ok, but 26 would be my preference

Reply:
Lets keep it for now.